src/tools/tool-options.h

```diff
@@ -91,36 +83,32 @@ struct ToolOptions : public Options {
              std::string("Disable ") + description, Arguments::Zero,
              [=](Options*, const std::string&) {
                hasFeatureOptions = true;
-               passOptions.features.set(feature, false);
                enabledFeatures.set(feature, false);
                disabledFeatures.set(feature, true);
              });
     return *this;
   }
 
-  void calculateFeatures(const Module& module) {
-    FeatureSet wasmFeatures;
-    bool wasmHasFeatures =
-        ModuleUtils::readFeaturesSection(module, wasmFeatures);
-
+  void calculateFeatures(Module& module) {
```

Comment:
how about "applyFeatures"? I think that makes it clear it applies them to the module.

Reply:
done.